chapters/overloaded.tex

```diff
@@ -60,15 +60,15 @@ \section{Matching Function}\label{matching-function}
 end f;
 \end{lstlisting}
 
-The vector $P$ indicates whether argument $m$ of \lstinline!f! has a default value (\lstinline!true! for default value, \lstinline!false! otherwise).
+The vector $P$ indicates whether argument $m$ of \lstinline!f! has a default value (true for default value, false otherwise).
 A call \lstinline!f($a_1$, $a_{2}$, $\ldots$, $a_{k}$, $b_{1}$ = $w_{1}$, $\ldots$, $b_{p}$ = $w_{p}$)! with distinct names $b_{j}$ is a valid match for the function \lstinline!f!, provided (treating \lstinline!Integer! and \lstinline!Real! as the same type)
 \begin{itemize}
 \item
   $A_{i}$ = typeOf($a_{i}$) for $1 \leq i \leq k$,
 \item
   the names $b_{j}$ = $u_{Q_{j}}$, $Q_{j} > k$, $A_{Q_{j}}$ = typeOf($w_{j}$) for $1 \leq j \leq p$, and
 \item
-  if the union of $\{i: 1 \leq i \leq k \}$, $\{Q_{j}: 1 \leq j \leq p\}$, and $\{m: P_{m} \text{ is \lstinline!true! and } 1 \leq m \leq n \}$ is the set $\{i: 1 \leq i \leq n\}$.
+  if the union of $\{i: 1 \leq i \leq k \}$, $\{Q_{j}: 1 \leq j \leq p\}$, and $\{m: P_{m} \text{ and } 1 \leq m \leq n \}$ is the set $\{i: 1 \leq i \leq n\}$.
```

Comment:
"is true" got lost completely. Is it intentional?

Reply:
Yes, I generally find it better to let Booleans speak for themselves, and here it also solves the problem of how to format the Boolean value.